contracts/refunded.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4; //teniamo la stessa versione di solidity?
```

Comment:
Be more consistent with the version of solidity. Sometime it's `0.8.0`, sometime it's `0.8.7`, and now it's `0.8.4`